server/controllers/profile.js

```diff
@@ -24,7 +24,7 @@ exports.editProfile = asyncHandler(async (req, res, next) => {
 // @desc Get user profile data
 // @access Private
 exports.loadProfile = asyncHandler(async (req, res, next) => {
-  const profile = await User.findById(req.user.id, "profile");
+  const profile = await Profile.findOne({ userId: req.params.userId });
```

Comment:
You might want to double check with the team if anyone is using this route for any other purpose (for example, obtaining the logged in user's profile)

Reply:
I change it is because it was finding a User object, not a Profile object. I just make it return a Profile, not a User.